client/src/pages/RoomPage/index.tsx

```diff
@@ -80,7 +80,10 @@ function RoomPage({ name }: { name: string }): JSX.Element {
   };
 
   const toggleMute = useCallback(() => {
-    updateSelfUserInfo({ mute: !selfUserInfo.mute });
+    updateSelfUserInfo({ 
+      mute: !selfUserInfo.mute, 
+      active: selfUserInfo.mute
+    });
```

Comment:
We'd still want to allow the status being active while mute is enabled just not the opposite (status being inactive while mute disabled). Right now, looks like if the user enables mute, it'll automatically also set the status to being inactive.